test/e2e/testdata/application.yaml

```diff
@@ -24,9 +24,11 @@ spec:
     name: TryOnce
   queue: root.default
   taskGroups:
-    - groupName: "test-task-0001"
-      minMember: 1
-      minResource:
-        cpu: "300m"
-        memory: "128Mi"
+    - schedulingPolicy: "TryReserve"
+    - groups:
```

Comment:
I liked better the previous structure of having just taskGroups field instead of having taskGroups, than groups. For me it seems redundant to have both fields. Can you please explain why it is better this structure than the previous one?

Reply:
I've changed it back. pls take a look.